docs/tutorials/basic_commands.md

```diff
@@ -28,21 +25,21 @@ functionalites of the tool. They are:
 * **disable**
 
 In this tutorial, we will go through all those commands to show how to properly use them.
-To achieve that, we will use a Xenial LXD container.
+To achieve that, we will use a Xenial Multipass virtual machine.
 
-## Creating the Xenial LXD container
+## Creating the Xenial Multipass virtual machine
 
-To test all of those commands, let's create a Xenial LXD container. Remember to set up LXD as
+To test all of those commands, let's create a Xenial Multipass VM. Remember to install multipass as
 mentioned on the [Prerequisites](#prerequisites) section. After that, just run the command:
 
 ```console
-$ lxc launch ubuntu-daily:xenial dev-x
+$ multipass launch xenial --name dev-x
 ```
 
-After running that, let's access the container by running:
+After running that, let's access the VM by running:
 
 ```console
-$ lxc shell dev-x
+$ multipass shell dev-x
 ```
 
```

Comment:
![image](https://user-images.githubusercontent.com/116022293/205892994-11650ac9-b1ce-4fbf-9eb5-e338b7d06963.png)

Reply:
I got stuck here - I managed to install multipass and start the VM, but it didn't like the "pro status" command.